hello.php

```diff
@@ -1,7 +1,17 @@
 <?php
 
-$string = "Hello %s\n";
+$hour = date('G');
 
-echo sprintf($string, $argv[1]);
+if ( $hour >= 5 && $hour <= 11 ) {
+    $greeting = "Good Morning";
+} else if ( $hour >= 12 && $hour <= 18 ) {
+    $greeting = "Good Afternoon";
```

Comment:
It's a good idea to always pass a DateTimezone to the DateTime class. Check the docs for more info http://php.net/manual/en/datetime.construct.php